lib/sww.js

```diff
@@ -2,6 +2,7 @@
 'use strict';
 
 var debug = 1 ? console.log.bind(console, '[ServiceWorkerWare]') : function(){};
```

Comment:
Can you remove the space before `var`, it makes me feel kind of uncomfortable. xP